tests/test_clean.py

```diff
@@ -8,9 +8,20 @@
 from wiktextract.wxr_context import WiktextractContext
 
 
-class WiktExtractTests(unittest.TestCase):
+class CleanTests(unittest.TestCase):
     def setUp(self) -> None:
         self.wxr = WiktextractContext(Wtp(), WiktionaryConfig())
+        self.NAMESPACE_DATA = {
```

Comment:
Probably don't need this? By default, `self.wxr.wtp` should load the en edition namespace data.

Reply:
I thought I had a problem, but that might have been because I was trying out git worktrees and it turned out that nothing worked because my venv would (of course) still point to the original editable repo and not the worktree copy-branch.

Reply:
Yeah, it was worktrees, and then I just "fixed" an issue that wasn't an issue.